tests/unit/ui/components/TestExpandableCard.js

```diff
@@ -32,24 +44,17 @@ describe(__filename, () => {
   });
 
   it('toggles when clicked', () => {
```

Comment:
I think this component could benefit from one other test that simulates two clicks and makes sure the state changes accordingly. Currently it only simulates one click which doesn't exercise the toggling part.